README.md

```diff
@@ -330,6 +330,17 @@ Then you can run this in the checkout root to reformat all Rust code:
 $ make rustfmt
 ```
 
+### Running tests
+
+* make check
+  Run all tests as defined in the directory.  Expensive tests are
+  suppressed.  The result of the tests for <filename>.el is stored in
+  <filename>.log.
+
+* make check-maybe
```

Comment:
Could you put the commands in backticks (e.g. `` `make check` ``), so they're clearly distinguishable form the text?